halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java

```diff
@@ -45,7 +45,7 @@
    *
    * @see Halconfig#halyardVersion
    */
-  String version = "nightly";
+  String version = "master-latest-unvalidated";
```

Comment:
This is a default/placeholder before we get have a proper "latest" alias that points to the latest validated Spinnaker. I know that for the next week before we release, this alias is just a QOL improvement that effectively points at what "nightly" pointed at.